spec/integration/validator_spec.rb

```diff
@@ -8,14 +8,20 @@
   include BoltSpec::Project
 
   around(:each) do |example|
-    with_project do
-      File.write((@project_path + 'inventory.yaml'), inventory.to_yaml)
+    with_project_directory(config: project_config, inventory: inventory) do |project_path|
```

Comment:
I think this can be `in_project`?

Reply:
Ah, I think I see - with `run_cli` you either need to specify a project or run with `--project`. Honestly now that we have the project helpers it seems like removing the `project` key from run_cli is best so we can standardize on using `with_project` and specifying `--project` on the CLI in cases where it needs it. But if that will take too long (maybe more than half a day) to do, we can do that separately and leave this as is for now.